LoRaEngine/example.env

```diff
@@ -38,7 +38,7 @@ FACADE_AUTH_CODE=yourauthcode
 RESET_PIN=7
 # not yet implemented LBS_SPI_SPEED=2
 LBS_TC_URI=ws://192.168.0.10:5000
-LBS_VERSION=1.0.6
+LBS_VERSION=2.1.0
```

Comment:
is 2.1.0 correct here? in the release notes we say `LoRaBasicsStationModule updated to Basics Station v2.0.6`

Reply:
The station binary is 2.0.6, our module versioning is 2.1.0 though 👍

Reply:
thanks for the clarification, resolved.